src/extension/options.js

```diff
@@ -198,8 +199,13 @@ function shareProject(i, evt) {
 }
 
 function editProject(i) {
-  getState(({ projects = [] }) => {
+  getState(async ({ projects = [] }) => {
     const project = projects[i];
+
+    const protectedProjects = await getConfig('local', 'hlxProtectedProjects') || {};
```

Comment:
We could reuse `hlxSidekickProjects` here for consistency, we already use the same property name in different storage areas:
- sync: standard project properties
- session: admin auth token
- local: project auth token